hc/front/views.py

```diff
@@ -27,6 +32,107 @@ def pairwise(iterable):
     return zip(a, b)
 
 
+def blogs(request, filter_by):
+    num = 3
+    category = Category.objects.all()
+    if int(filter_by) > 0:
+        category_id = Category.objects.get(pk=filter_by).id
+        stories = Blog_post.objects.filter(category=category_id)
+    else:
+        stories = Blog_post.objects.all()
+    list_of_stories = [story for story in stories]
+    blogs = [list_of_stories[i:i+num] for i in range(0, len(list_of_stories), num)]
+    ctx = {
+        'category':category,
+        'blogs':blogs
+    }
+    return render(request, "front/blog_posts.html", ctx)
+
+
+def create_blog(request):
+    form = CreateBlogPost(request.POST)
+    category_form = CreateCategory(request.POST)
+    if "new_category" in request.POST:
+        if category_form.is_valid():
+            name = category_form.cleaned_data['category']
+            ctg = Category(name = name)
+            ctg.save()
+    elif "create_blog" in request.POST:
+        if form.is_valid():
+            title = request.POST['title'] 
+            blog = form.cleaned_data['content']
+            selected_category = request.POST['category_name']
+            category = Category.objects.get(name=selected_category)
+            published = timezone.now()
+            user = request.user
+            blog = Blog_post(title=title, content=blog, category=category,
+                             published=published, user=user)
+            blog.save()
+            messages.add_message(request, messages.INFO, 'Successfully created blog')
```

Comment:
Be consistent with the quotation marks you choose to use. either go with double or single.